### PR TITLE
Displayed all the property paths on incoming and outgoing

### PR DESCRIPTION
**Pull request by umbreak**

### CODE REVIEW

File: src/main/resources/blazegraph/incoming.txt [hunk 1]
```diff
@@ -28,8 +36,6 @@ WHERE {
   {
     SELECT *
     WHERE { INCLUDE %resultSet }
-    ORDER BY ?s ?property
-    LIMIT {size}
-    OFFSET {offset}
-   }
+    ORDER BY ?_createdBy ?s ?paths LIMIT 20 OFFSET 0
```
Comment (bogdanromanx) [inline src/main/resources/blazegraph/incoming.txt]: limit and offset template?

File: src/main/resources/blazegraph/outgoing_include_external.txt [hunk 2]
```diff
@@ -1,46 +1,50 @@
...
-}
\ No newline at end of file
+    SELECT (COUNT(?s) AS ?total)
+    WHERE { INCLUDE %resultSet }
+  }
+  UNION
+  {
+    SELECT *
+    WHERE { INCLUDE %resultSet }
+    ORDER BY ?_createdBy ?s ?paths LIMIT 20 OFFSET 0
```
Comment (bogdanromanx) [inline src/main/resources/blazegraph/outgoing_include_external.txt]: limit and offset template?

File: src/main/resources/blazegraph/outgoing_scoped.txt [hunk 3]
```diff
@@ -1,40 +1,44 @@
...
-}
\ No newline at end of file
+    SELECT (COUNT(?s) AS ?total)
+    WHERE { INCLUDE %resultSet }
+  }
+  UNION
+  {
+    SELECT *
+    WHERE { INCLUDE %resultSet }
+    ORDER BY ?_createdBy ?s ?paths LIMIT 20 OFFSET 0
```
Comment (bogdanromanx) [inline src/main/resources/blazegraph/outgoing_scoped.txt]: limit and offset template?

**Comment by umbreak** [comment]
Of course I have run them. Most of the part of the query is happening inside the scope of a resource, where there are very few triples.

The part which is not happening on the scope of the graph is after already selecting the links, which shouldn't be excessive. A regular payload is not supposed to have 1000 of links.

So I think it if more than good enough.

**Pull request merged**